KioskTests/App/RAC/RACFunctionsTests.swift

```diff
@@ -0,0 +1,124 @@
+import Quick
+import Nimble
+import Kiosk
+import ReactiveCocoa
+
+class RACFunctionsTests: QuickSpec {
+    override func spec() {
+        describe("email address check") {
+            it("requires @") {
+                let valid = stringIsEmailAddress("ashartsymail.com") as Bool
+                expect(valid) == false
+            }
+
+            it("requires name") {
+                let valid = stringIsEmailAddress("@artsymail.com") as Bool
+                expect(valid) == false
+            }
+
+            it("requires domain") {
+                let valid = stringIsEmailAddress("ash@.com") as Bool
+                expect(valid) == false
+            }
+
+            it("requires tld") {
+                let valid = stringIsEmailAddress("ash@artsymail") as Bool
+                expect(valid) == false
+            }
+
+            it("validates good emails") {
+                let valid = stringIsEmailAddress("ash@artsymail.com") as Bool
+                expect(valid) == true
+            }
+        }
```

Comment:
you may find shared assertions in quick can reduce the LOC for these

https://github.com/Quick/Quick/blob/master/Documentation/SharedExamples.md#reducing-test-boilerplate-with-shared-assertions